static/map.js

```diff
@@ -34,10 +40,10 @@ function pad(num, size) {
     return s;
 }
 
-document.getElementById('pokemon-checkbox').checked = getFromStorage("displayPokemons", "true");
-document.getElementById('gyms-checkbox').checked = getFromStorage("displayGyms", "true");
-document.getElementById('coverage-checkbox').checked = getFromStorage("displayCoverage", "true");
-
+document.getElementById('pokemon-checkbox').checked = getFromStorage("displayPokemons", true);
+document.getElementById('gyms-checkbox').checked = getFromStorage("displayGyms", true);
+document.getElementById('coverage-checkbox').checked = getFromStorage("displayCoverage", true);
```

Comment:
In these 3 lines the default value was set to the string "true", which evaluates to the boolean true when put into .checked. This works but is not correct.

Instead, I set the default value to the boolean true for correctness.